### PR TITLE
#152 apply styling to explorer

### LINKED ISSUE #152: Apply styling to explorer
Take the design from inVision and apply that to the explorer table.

## Expected Behaviour
Look nice

## Current Behaviour
Looks nice already - will look better

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #152 

## Description
Makes the table look more like the mock up. Could be more beautiful, but then, couldn't we all?
- Has the table height more adjusted to the max available in the body area, with a scroll for the results. 
- UC icon updated with the version from Pete.
- header and footer heights have been reduce a little
- footer changed from `Drawer` to `Paper` as the drawer causes a few issues
- `Drawer` removed from project ( no longer used )
- theme updated to include some customised options
- background colour of page updated to give contrast to table

### Documentation:

Tick below if one of the following applies:

1. Documentation has been added to cover changes introduced by this PR.
2. This PR requires no changes to documentation.

- [ ] Documentation OK.

### Tests:

Tick below if one of the following applies:

1. Tests have been added to cover changes introduced by this PR.
2. This PR requires no new tests.

- [ ] Tests OK.

### CODE REVIEW

**Comment by mark-prins** [comment]
This is branched off #172  - that one should be merged first to make review of this one easier.

**Comment by mark-prins** [comment]
@wlthomson & @katherine-sussol - can you have a look at this one please? 
It will need to go in before #177

**Review by katherine-sussol — APPROVED** [review]
Looks a little wonky after I merged master into it (mostly the empty white space underneath the header), but it looked really good until then! Approving so we can fix it in a separate issue!

File: unified-codes/libs/ui/src/lib/components/inputs/molecules/SearchBar.tsx
Comment (katherine-sussol) [inline unified-codes/libs/ui/src/lib/components/inputs/molecules/SearchBar.tsx:37]: Also - not a criticism but me wondering when it's good to put styles where 😕 . Inline for small things?

File: unified-codes/apps/web/src/components/templates/Explorer.tsx
Comment (mark-prins) [reply unified-codes/apps/web/src/components/templates/Explorer.tsx:124]: oh. the overflow - I've fixed that twice now in separate PRs. I'll merge this and try to get everything up to date.

File: unified-codes/libs/ui/src/lib/components/inputs/molecules/SearchBar.tsx
Comment (mark-prins) [reply unified-codes/libs/ui/src/lib/components/inputs/molecules/SearchBar.tsx:37]: I think inline is not good, no idea who did this!!

**Pull request merged**
